packages/fcl/src/current-user/exec-service/strategies/http-post.js

```diff
@@ -3,10 +3,9 @@ import {normalizePollingResponse} from "../../normalize/polling-response"
 import {normalizeLocalView} from "../../normalize/local-view"
 import {poll} from "./utils/poll"
 import {execLocal} from "../exec-local"
-import {configLens} from "../../../config-utils"
 import {VERSION} from "../../../VERSION"
 
-export async function execHttpPost(service, signable, opts = {}) {
+export async function execHttpPost(service, signable, opts = {}, config = {}) {
```

Comment:
Do you need the defaults if defaults are added in execService?
`execHttpPost(service, signable, opts, config)`